data/hails.txt

```diff
@@ -3112,6 +3112,62 @@ phrase "hostile militia"
 		"."
 		"!"
 
+phrase "hostile militia"
+	word
+		""
+		"You! "
+		"You there! "
+		"Hey, you! "
+	word
+		"The"
+		"That"
+		"Our"
+		"The local"
+	word
+		" prison"
+		" jail"
+		" penitentiary"
+		" correctional facility"
+		" reformatory"
+	word
+		" on "
+		" at "
+		" over on "
+		" over at "
+		" out on "
+		" out at "
+	word
+		"Bourne"
+		"Spica"
+		"Rust"
+		"Starcross"
+		"Skymoot"
+		"Thunder"
+		"Zug"
+		"Flood"
+		"Longjump"
+		"Glaze"
+		"Deep"
+		"Clink"
+		"Winter"
+		"Solace"
+		"Twinstar"
+		"Sundrinker"
+		"Trinken"
```

Comment:
hmm...

Reply:
Had you been drinken when you wrote that?

Reply:
skål